Library/Formula/ncutil.rb

```diff
@@ -0,0 +1,15 @@
+class Ncutil < Formula
+  desc "Modify OS X's Notification Center: Add/remove apps, get/set alert styles, suppress notifications, etc."
+  homepage "https://github.com/jacobsalmela/NCutil"
+  url "https://github.com/jacobsalmela/NCutil/archive/2.4.tar.gz"
+  version "2.4"
+  sha256 "ebea1fe1163e401dd4e7833416001dbf9d60614237700755b55b46d0abf423bc"
+
+  def install
+	bin.install "NCutil.py" => "ncutil"
+  end
+
+  test do
+	system "#{bin}/ncutil", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!